lib/l10n/app_es.arb

```diff
@@ -0,0 +1,539 @@
+{
+  "appName": "Aves",
+  "welcomeMessage": "Bienvenido a Aves",
+  "welcomeOptional": "Opcional",
+  "welcomeTermsToggle": "Acepto los términos y condiciones",
+  "itemCount": "{count, plural, =1{1 elemento} other{{count} elementos}}",
+  
+  "timeSeconds": "{seconds, plural, =1{1 segundo} other{{seconds} segundos}}",
+  "timeMinutes": "{minutes, plural, =1{1 minuto} other{{minutes} minutos}}",
+  
+  "applyButtonLabel": "APLICAR",
+  "deleteButtonLabel": "BORRAR",
+  "nextButtonLabel": "SIGUIENTE",
+  "showButtonLabel": "MOSTRAR",
+  "hideButtonLabel": "OCULTAR",
+  "continueButtonLabel": "CONTINUAR",
+
+  "cancelTooltip": "Cancelar",
+  "changeTooltip": "Cambiar",
+  "clearTooltip": "Limpiar",
+  "previousTooltip": "Anterior",
+  "nextTooltip": "Siguiente",
+  "showTooltip": "Mostrar",
+  "hideTooltip": "Ocultar",
+  "actionRemove": "Remover",
+  "resetButtonTooltip": "Restablecer",
+
+  "doubleBackExitMessage": "Presione “atrás” nuevamente para salir.",
```

Comment:
In English curly quotes are used, like “content”.
In French the angular quotes are used (with unbreakable spaces \u00A0 inside) like « content ».

I noticed you kept the quotation marks used in English. Is it intentional? From what I know both the curly and angular quotes are used in Spanish, but I don't know which ones are recommended.

Reply:
In formal speech and according to Royal Spanish Academy the angular marks have to be used, but in practice nobody uses them outside serious academic works.
The LA Spanish full keyboard layout only features straight simple and double marks, the angulars are nowhere to be found on the layout, instead, ALT+174/175 inserts them «»

The curly marks are uncommon when it comes to typing on a computer or electric typewriter as, once again, they're not present on the keyboard, they're used for cursive handwriting or at least that's what I've always seen.

Reply:
Thanks for the explanation! I'm no expert but I care about typography and following recommended practices, regardless of keyboard limitations. In other words, I try to go with book typography, not SMS typography. I'll apply the RSA recommendation.

Reply:
If so the straight marks on strings 28, 136, 138, 140, 142, and 369 would have to be replaced with angulars. I think those are all. I see the request is closed, should I do it anyway?

At one point I thought of replacing the curly marks with straights but I have a friend who's a coder and told me that it'd cause issues to write something like `""word"", ` without a line containing an } inbetween, not sure why exactly but the program would loop.

Also I just noticed at line 506 _Alojado_ has an r making it _Alojador,_ that's wrong.

Reply:
I'm taking care of it now. Thanks for the heads-up on l.506, I'll fix it.

Reply:
Typos happen because I'm not used to Sublime Text yet, it either removes too much or leaves characters that shouldn't be there. 
I used to work using Notepad but the white background burns my eyes if I work during the night for more than 5 minutes and also has tons of formatting problems.